packages/apps/human-app/server/src/common/config/environment-config.service.ts

```diff
@@ -61,35 +61,7 @@ export class EnvironmentConfigService {
   get corsAllowedHeaders(): string {
     return this.configService.get<string>(
       'CORS_ALLOWED_HEADERS',
-      DEFAULT_CORS_ALLOWED_HEADERS
+      DEFAULT_CORS_ALLOWED_HEADERS,
     );
   }
-  conditionallyReturnMandatoryEnvVariable<T>(envName: string): T {
-    const value = this.configService.get<T>(envName);
-    if (!value) {
-      throw new EnvironmentVariableMissingError(envName);
-    }
-    return value;
-  }
-  checkMandatoryConfig(): void {
```

Comment:
Why you removed this method: we want to know that config is missing as fast as possible not in the moment of actually needing it

Reply:
fix: 8cbdb5d1 (the only place we will validate the required environment variables is in ConfigModule.forRoot, as it runs before the modules are loaded, this ensures that we are informed about any missing configurations as fast as possible)